packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -241,7 +241,10 @@ function isSafeUse(node: TSESTree.Node): boolean {
       return parent.tag === node;
 
     case AST_NODE_TYPES.UnaryExpression:
-      return parent.operator === 'typeof';
+      // the first case is safe for obvious
+      // reasons. The second one is also fine
+      // since we're returning something falsy
+      return ['typeof', '!'].includes(parent.operator);
```

Comment:
while we're at it, could we also whitelist both `void` and `delete`?
They're both equally safe for obvious reasons

Reply:
Yeah sure. One moment.

Reply:
Oops. Guess I forgot to mention this yesterday, but the extra whitelisted operators were added in the most recent commit. So should be good to go pending any further recommendations.